packages/svelte-vscode/package.json

```
@@ -396,6 +396,17 @@
                 }
             }
         },
+        "configurationDefaults": {
+            "[svelte]": {
+                "editor.defaultFormatter": "svelte.svelte-vscode"
+            },
+            "workbench.editor.customLabels.patterns": {
+                "src/routes/+[layout|page|server]*": "/${filename}.${extname}",
+                "src/routes/*/+[layout|page|server]*": "/${dirname}/${filename}.${extname}",
+                "src/routes/*/*/**/+[layout|page|server]*": ".../${dirname}/${filename}.${extname}",
```

this didn't work for me in a monorepo because `src` was contained within another directory. I think we might need a leading `**/`
```suggestion
                "**/src/routes/+[layout|page|server]*": "/${filename}.${extname}",
                "**/src/routes/*/+[layout|page|server]*": "/${dirname}/${filename}.${extname}",
                "**/src/routes/*/*/**/+[layout|page|server]*": ".../${dirname}/${filename}.${extname}",
```